shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -229,15 +250,25 @@ public AccessibilityNodeInfo createAccessibilityNodeInfo(int virtualViewId) {
                 result.addAction(AccessibilityNodeInfo.ACTION_SCROLL_BACKWARD);
             }
         }
-
+        if (object.hasFlag(Flag.IS_LIVE_REGION)) {
+            result.setLiveRegion(View.ACCESSIBILITY_LIVE_REGION_POLITE);
+        } else if (object.hadFlag(Flag.IS_LIVE_REGION)) {
+            mLiveRegions.remove(object.id);
```

Comment:
Why is it not added to mLiveRegions in the if branch?

Reply:
Unfortunately I had a bug and it was covering up why this wasn't working.  We can only get a live region update _after_ Talkback is aware of the fact that the node in question is a live region.  Because I was mistakenly not marking it sent after the first pass, a later build would trigger the update correctly.